llm/llama-2/chatbot-hf.yaml

```diff
@@ -19,6 +19,7 @@ setup: |
   pip install git+https://github.com/lm-sys/FastChat.git
   # Need the latest transformers to support 70B model
   pip install git+https://github.com/huggingface/transformers.git
```

Comment:
We should be able to remove these lines as they are installed in `fschat[model_worker,webui]` already. ; )
```suggestion
```

Reply:
Yes, I just updated it and it looks neat now.